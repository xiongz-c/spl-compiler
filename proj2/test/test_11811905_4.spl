struct testStruct {} a;

int func() {
    return 10;
}

int test_11811806_4() {
    float v = func() / 2;
    char a = 'a';
    if (func() > a) {
        return func();
    }else if ( func() > a ){
        return 0;
    }
}