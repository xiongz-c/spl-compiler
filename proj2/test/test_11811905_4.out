Error type 5 at Line 8: unmatching type on both sides of assignment
Error type 7 at Line 10: binary operation on non-number variables
Error type 7 at Line 12: binary operation on non-number variables
